Jenkinsfile

```diff
@@ -30,6 +31,7 @@ pipeline {
           docker push "$IMG_TAG"
           docker push "$BASE_PUSH_TARGET:devel-$GIT_BRANCH-$BASE_OS"
           docker push "$BASE_PUSH_TARGET:devel-$GIT_BRANCH-n-$BUILD_NUMBER-id-$BUILD_ID-$BASE_OS"
```

Comment:
Do we need this tag pushed as well or is it sufficient to just push the new one with the added timestamp?